assets/javascripts/discourse/templates/explorer/badge.raw.hbs

```diff
@@ -3,4 +3,4 @@
    class="user-badge {{badge.badgeTypeClassName}}"
    title="{{badge.displayDescription}}"
    data-badge-name="{{badge.name}}">{{icon-or-image badge.icon}}
-<span class="badge-display-name">{{badge.displayName}}</span></a>
```

Comment:
I think instead of using `name` which might not always be user friendly, it would be better to add the [display_name](https://github.com/discourse/discourse/blob/956f8492507f6004272b1e6900f3e56c5c0f41a1/app/models/badge.rb#L256-L258) property to the [`small_badge_serializer`](https://github.com/discourse/discourse-data-explorer/blob/master/app/serializers/data_explorer/small_badge_serializer.rb).

Reply:
You're right, I've fixed this.